### ISSUE TITLE
licensing problems

### ISSUE THREAD

**Issue by draeath** [mlheur/SKSE-SE#4]
Here's the text of the original licenses, as per 1.07.03

for src/common:

```
This license applies to all of the files in src/common:

Copyright (c) 2006-2011 Ian Patterson

This software is provided 'as-is', without any express or implied
warranty. In no event will the authors be held liable for any damages
arising from the use of this software.

Permission is granted to anyone to use this software for any purpose,
including commercial applications, and to alter it and redistribute it
freely, subject to the following restrictions:

   1. The origin of this software must not be misrepresented; you must not
   claim that you wrote the original software. If you use this software
   in a product, an acknowledgment in the product documentation would be
   appreciated but is not required.

   2. Altered source versions must be plainly marked as such, and must not be
   misrepresented as being the original software.

   3. This notice may not be removed or altered from any source
   distribution.
```

and for src/skse:

```
These notes apply to all of the files in src/skse:

Due to continued intentional copyright infringement and total disrespect for modder etiquette, the Skyrim Online team is explicitly disallowed from using any of these files for any purpose.
```

What's the problem?: All three claims for src/common are broken. Also, src/skse does not actually issue a license but only prohibits a specific use. This generally means "all rights reserved" as they haven't applied a specific license.

I'm asking the team (via email) to swing by here and either bless this, or clarify things.

**Comment by draeath** [comment]
To clarify claim 1 is only tenuously broken, as you've put your own name in your own license file but make no mention of the original author... but I believe 2 and 3 are plainly broken.

Sorry to be the guy :( hopefully the team comes back and says 'go ahead!'

**Comment by mlheur** [comment]
Thanks for the head's up - I thought I saw the MIT license somewhere else in the source base but I can't find it now. That could've been from bsaopt or xEdit since I was bouncing around from a few code bases.

I'm happy to delete the repo, if the silverlight team wants me to. I was going to make it private but I'm too cheap to pay for github.

**Comment by ianpatt** [comment]
Correct, SKSE's source code has two parts. Everything in common is released under a permissive license, and everything else is not under a permissive license. The only explicit license term for the code is due to some of the most ugly "mod drama" I have ever needed to deal with.

The main code is not under a permissive license because forks would be extremely damaging to the user base.

**Comment by draeath** [comment]
@mlheur well, we'll see what the decision is. BTW, if you want private repos but don't want to fork over cash, Atlassian's [Bitbucket](https://bitbucket.org/product) will do. Also [AWS' CodeCommit is free for light usage](https://aws.amazon.com/codecommit/pricing/), and stupid cheap for otherwise.
